Makefile

```diff
@@ -85,7 +85,7 @@ test: ## Run the test suite
 # ------------------------------
 
 .PHONY: check
-check: check-format check-types
+check: check-format check-types ## Run various checks on project files
```

Comment:
I think this lines missing `check-code-coverage`

Reply:
Yes, but I think this is mandatory:

https://github.com/mirego/ember-boilerplate/blob/master/.github/workflows/ci.yaml#L31

Reply:
Ah yes, tests must be run first 🤦‍♂️